harness/determined/cli/experiment.py

```diff
@@ -9,12 +9,15 @@
 from argparse import FileType, Namespace
 from pathlib import Path
 from pprint import pformat
-from typing import Any, Dict, List, Optional, Set, Tuple
+from typing import Any, Dict, Iterable, List, Optional, Set, Tuple
 
 import tabulate
 
-import determined.common
+import determined
```

Comment:
Can we stick with the cannonical `import determined as det`?

Reply:
Since we use an "underscored" thing from `import determined`, which doesn't look like a canonical user-facing thing, I've switched it to a direct function import.